R/utils.R

```diff
@@ -5,6 +5,47 @@
 #' @export
 #'
 colfun <- function() {
+  control_palettes <- data.frame(
+    name = c(
+      "Blue",
+      "Purple",
+      "Pink",
+      "Orange",
+      "Black",
+      "Grey 4",
+      "Grey 3",
+      "Grey 2",
+      "Grey 1",
+      "Secondary Blue",
+      "Secondary Green"
+    ),
+    hex = c(
+      "#354B96",
+      "#5E366E",
+      "#CF004D",
+      "#EE8000",
+      "#1C1C1B",
+      "#555555",
+      "#888888",
+      "#BCBCBC",
+      "#ECEDED",
+      "#4EADD0",
+      "#96BA39"
+    ),
+    order = c(
+      1,
+      2,
+      3,
+      4,
+      5,
+      6,
+      7,
+      8,
+      9,
+      10,
+      11
+    )
+  )
   fig2_colors <- c("#00AFBB", "#FFDB6D")
```

Comment:
We may consider renaming these to avoid the use of figX in the codes.

Reply:
for the color function we can leave as is for now as we are generating the figures for the manuscript. LWe can consider the colors to be controlled via an argument in the function?